src/containerapp/azext_containerapp/custom.py

```diff
@@ -0,0 +1,954 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.azclierror import (RequiredArgumentMissingError, ResourceNotFoundError, ValidationError)
+from azure.cli.core.commands.client_factory import get_subscription_id
+from azure.cli.core.util import sdk_no_wait
+from knack.util import CLIError
+from knack.log import get_logger
+from msrestazure.tools import parse_resource_id, is_valid_resource_id
+from msrest.exceptions import DeserializationError
+
+from ._client_factory import handle_raw_exception
+from ._clients import ManagedEnvironmentClient, ContainerAppClient
+from ._sdk_models import *
```

Comment:
Same question here - are all the models here needed?

Reply:
Moved this to inside deserialize function, so SDK models are only used when customer uses YAML and deserialization is needed. Also changed from wildcard to only import ContainerApp